### PR TITLE
[CARBONDATA-3791] Correct spelling, link and ddl in SI and MV  Documentation

### PR DESCRIPTION
**Pull request by Indhumathi27**
### Why is this PR needed?
 Correct spelling, link and ddl in SI and MV  Documentation
 
 ### What changes were proposed in this PR?
Fixed spelling, link and ddl in SI and MV  Documentation
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2910/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1193/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1199/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2918/

File: docs/index/bloomfilter-index-guide.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ User can create BloomFilter Index using the Create Index DDL:
   PROPERTIES ('BLOOM_SIZE'='640000', 'BLOOM_FPP'='0.00001', 'BLOOM_COMPRESS'='true')
   ```
 
-Here, (name,id) are INDEX_COLUMNS. Carbondata will generate BloomFilter index on these columns. Queries on these columns are usually like 'COL = VAL'.
+Here, (name,id) are INDEX_COLUMNS. Carbondata will generate BloomFilter index on these columns. Queries on these columns are usually like `'COL = VAL'`.
```
Comment (chetandb) [inline docs/index/bloomfilter-index-guide.md:99]: 'COL = VAL' is fine. Why need to add another ' at start and end

File: docs/index/lucene-index-guide.md [hunk 2]
```diff
@@ -85,20 +85,20 @@ index_columns(String Columns) given in CREATE statement which contains informati
 location of index_columns. These index files will be written inside a folder named with index name
 inside each segment folders.
```
Comment (chetandb) [inline docs/index/lucene-index-guide.md:86]: Change to "inside each segment folder"

File: docs/index/bloomfilter-index-guide.md (hunk 1 above)
Comment (Indhumathi27) [reply docs/index/bloomfilter-index-guide.md:99]: to highlight it in the document

File: docs/index/secondary-index-guide.md [hunk 3]
```diff
@@ -159,7 +159,7 @@ with in WHERE clause.
   ```
   SELECT country, sex from sales where NI(user_id = 'xxx')
   ```
-The above query ignores column user_id from secondary index and fetch data from main table.
+The above query ignores column `user_id` from secondary index and fetch data from the main table.
```
Comment (chetandb) [inline docs/index/secondary-index-guide.md:162]: Change fetch to fetches

File: docs/mv-guide.md [hunk 4]
```diff
@@ -207,15 +207,15 @@
       materialized view, if not, the operation is allowed, otherwise operation will be rejected by
       throwing exception.
    3. Partition management command: `ALTER TABLE ADD/DROP PARTITION`. Note that dropping a partition
-      will be allowed only if partition is participating in all indexes associated with fact table.
+      will be allowed only if partition is participating in all materialized views associated with fact table.
```
Comment (chetandb) [inline docs/mv-guide.md:210]: "if partition" can be changed to "if the partition"

Comment (Indhumathi27) [reply docs/mv-guide.md]: changed

File: docs/index/secondary-index-guide.md (hunk 3 above)
Comment (Indhumathi27) [reply docs/index/secondary-index-guide.md]: changed

File: docs/index/lucene-index-guide.md (hunk 2 above)
Comment (Indhumathi27) [reply docs/index/lucene-index-guide.md]: changed

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2930/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1212/

File: docs/index/index-management.md [hunk 5]
```diff
@@ -51,15 +51,15 @@ Currently, there are 3 Index implementations in CarbonData.
 
 There are two kinds of management semantic for Index.
 
-1. Automatic Refresh: Create index without `WITH DEFERRED REBUILD` in the statement, which is by default.
-2. Manual Refresh: Create index with `WITH DEFERRED REBUILD` in the statement
+1. Automatic Refresh: Create index without `WITH DEFERRED REFRESH` in the statement, which is by default.
+2. Manual Refresh: Create index with `WITH DEFERRED REFRESH` in the statement
```
Comment (akashrn5) [inline docs/index/index-management.md:55]: please rewrite the 1 and 2 point for manual and automatic refresh, its not completely meaningfull.

File: docs/index/index-management.md [hunk 6]
```diff
@@ -78,7 +78,7 @@ We do recommend you to use this management for index.
 
 ### Manual Refresh
 
-When user creates a index specifying manual refresh semantic, the index is created with status *disabled* and query will NOT use this index until user can issue REFRESH INDEX command to build the index. For every REFRESH INDEX command, system will trigger a full rebuild of the index. After rebuild is done, system will change index status to *enabled*, so that it can be used in query rewrite.
+When user creates a index specifying manual refresh semantic, the index is created with status *disabled* and query will NOT use this index until user can issue REFRESH INDEX command to build the index. For every REFRESH INDEX command, system will trigger a full refresh of the index. After a refresh is done, system will change index status to *enabled*, so that it can be used in query rewrite.
```
Comment (akashrn5) [inline docs/index/index-management.md:81]: ` After a refresh is done` to `Once the refresh operation is finished`

File: docs/index/lucene-index-guide.md [hunk 7]
```diff
@@ -83,28 +84,28 @@ It will show all Indexes created on main table.
...
 
-A system level configuration carbon.lucene.compression.mode can be added for best compression of
+A system level configuration `carbon.lucene.compression.mode` can be added for best compression of
 lucene index files. The default value is speed, where the index writing speed will be more. If the
 value is compression, the index file size will be compressed.
 
 ## Querying data
 As a technique for query acceleration, Lucene indexes cannot be queried directly.
-Queries are to be made on main table. when a query with TEXT_MATCH('name:c10') or 
+Queries are to be made on main table. When a query with TEXT_MATCH('name:c10') or
```
Comment (akashrn5) [inline docs/index/lucene-index-guide.md:95]:
```suggestion
Queries are to be made on the main table. When a query with TEXT_MATCH('name:c10') or 
```

File: docs/mv-guide.md [hunk 8]
```diff
@@ -300,19 +300,19 @@
  granularity provided during creation and stored on each segment.
  
  **NOTE**:
-   1. Single select statement cannot contain time series udf(s) neither with different granularity
+   1. Single select statements cannot contain time series udf(s) neither with different granularity
```
Comment (akashrn5) [inline docs/mv-guide.md:303]: i think this change is not required

File: docs/mv-guide.md [hunk 9]
```diff
@@ -334,10 +334,10 @@
    ```
 
  Then, the above query can be rolled up from materialized view 'agg_sales', by adding hour
- level time series aggregation on minute level aggregation. Users can fire explain command
- to check if query is rolled up from existing materialized view.
+ level time series aggregation on minute level aggregation. Users can fire the EXPLAIN command
```
Comment (akashrn5) [inline docs/mv-guide.md:337]: put EXPLAIN  in the quotes of code

File: docs/index/index-management.md (hunk 5 above)
Comment (Indhumathi27) [reply docs/index/index-management.md:55]: Description is already given below

Comment (akashrn5) [reply docs/index/index-management.md:55]: description is there, please change the existing sentence to more meaningful

Comment (Indhumathi27) [reply docs/index/index-management.md]: done

File: docs/index/index-management.md (hunk 6 above)
Comment (Indhumathi27) [reply docs/index/index-management.md]: done

File: docs/index/lucene-index-guide.md (hunk 7 above)
Comment (Indhumathi27) [reply docs/index/lucene-index-guide.md]: done

File: docs/mv-guide.md (hunk 8 above)
Comment (Indhumathi27) [reply docs/mv-guide.md]: done

File: docs/mv-guide.md (hunk 9 above)
Comment (Indhumathi27) [reply docs/mv-guide.md]: done

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2940/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1222/

**Comment by Indhumathi27** [comment]
@akashrn5 Please review and merge

File: docs/mv-guide.md [hunk 10]
```diff
@@ -207,15 +207,15 @@
       materialized view, if not, the operation is allowed, otherwise operation will be rejected by
       throwing exception.
    3. Partition management command: `ALTER TABLE ADD/DROP PARTITION`. Note that dropping a partition
-      will be allowed only if partition is participating in all indexes associated with fact table.
+      will be allowed only if the partition is participating in all materialized views associated with fact table.
```
Comment (akashrn5) [inline docs/mv-guide.md:210]:
```suggestion
      will be allowed only if the partition columns of fact table is participating in all of the table's materialized views.
```

Comment (Indhumathi27) [reply docs/mv-guide.md]: done

**Comment by akashrn5** [comment]
LGTM

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1228/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2946/

**Pull request closed without merging**